### PR TITLE
🤖 HTML is for crawlers

### PR DESCRIPTION
**Pull request by 0xjjpa**

### CODE REVIEW

**Review by lkngtn — COMMENTED** [review]
I think setting the expectation that people provide a markdown link their profile is reasonable.

**Review by lkngtn — APPROVED** [review]

File: member_registry.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+| [@stellarmagnet](https://keybase.io/stellarmagnet) | `0xeBAC3108C58Bd4dD48a8110B7750480Cea7bC3Fb`  |
+| [@jjperezaguinaga](https://keybase.io/jjperezaguinaga) | `0x016b4De659f33f27e0C9281E41C5EBd6942c6f61` |
+| [@bingen](https://keybase.io/bingen) | `0x6F82dfD56A399390EFe5140d8E0390B200Dbb935` |
+| [@imestin](https://keybase.io/imestin)  | `0xF971b59792c8D122602b37E00d310fE069A03619` |
+| [@louisgrx](https://keybase.io/louisgrx) | `0xbe40DCADB6bF4De1c7810EcA3504E5f3BEA9cF35` |
+| [@maxsemenchuk](https://keybase.io/maxsemenchuk) | `0xA1c4753Ba0abb8230765660c662CCdE5B801Fb54` |
+| [@rperez89](https://keybase.io/rperez89)  | `0x593e1F9809658d0c92e9f092cF01Aad7D0d734f3` |
+| [@ro5s](https://keybase.io/ro5s) | `0x1409f15D77F28882C220860d5F534A1A7a9F3481` |
+| [@schmidsi](https://keybase.io/schmidsi) | `0xe98C357458C89198575a32462760402CaB2A9a4D` |
+| [@goliat](https://keybase.io/goliat) | `0x3E0B02c9f2c5107c2747fCb0E68d997bE2fa0F7f`
```
Comment (stellarmagnet) [inline member_registry.md]: Missing a pipe at the end

Comment (stellarmagnet) [reply member_registry.md]: I guess that may not have been your fault - do i just fix it? :)

Comment (0xjjpa) [reply member_registry.md]: No worries, I went ahead and fixed it ☺️

**Review by bingen — APPROVED** [review]

**Comment by lkngtn** [comment]
I think maybe we should hold off on this one and clear all the other membership request, then update this PR?

**Comment by 0xjjpa** [comment]
Agreed! Going to add a `don’t merge` label for now.

**Comment by 0xjjpa** [comment]
Going forward before more people apply with the old file!

**Pull request merged**